src/lib/expansion/expansion.md

```diff
@@ -0,0 +1,81 @@
+`<md-expansion-panel>` provides a way to show and hide lightweight content, by collapsing and expanding a view with a nice animation.
+
+<!-- example(expansion-overview) -->
+
+The `expanded` input allows to choose among the collapsed or expanded state. 
+
+Each expansion panel has a header and an action sections. The header section is always visible at the top of the component and contains a title and a description subsections. The action section is fixed at the bottom, being visible when the expansion is in expanded state.
```

Comment:
I would start a new section here like 
```markdown
### Expansion panel content
```
And then talk about the header, actions, title, and description.